mindee/parsing/standard/base.py

```diff
@@ -167,7 +167,7 @@ def int_to_string(value: Optional[int]) -> str:
     if value is None:
         return ""
 
-    return f"{value}"
+    return f"{value}.00"
```

Comment:
an int should never be rendered as a float

Reply:
Wow, I remember doing that, but it feels like a fever dream now :woozy_face: